tools/scope_download_classification.py

```diff
@@ -1,82 +1,177 @@
+#!/usr/bin/env python
 import argparse
-import csv
-import json
+import json as JSON
 import pandas as pd
 from penquins import Kowalski
 from time import sleep
-from scope.fritz import save_newsource, api
+from scope.fritz import api
+import warnings
+import numpy as np
 
 
-def download_classification(file, gloria, group_ids):
+def download_classification(file: str, gloria, group_ids: list, token: str):
     """
-    Upload labels to Fritz
-    :param file: file containing labels (csv)
+    Download labels from Fritz
+    :param file: CSV file containing obj_id column or "parse" to query by group ids (str)
     :param gloria: Gloria object
-    :param group_ids: group id on Fritz for upload target location (list)
+    :param group_ids: target group ids on Fritz for download (list)
+    :param token: Fritz token (str)
     """
-    # create new empty classification column
-    file["classification"] = pd.NaT
-
-    for index, row in file.iterrows():
-        if index > 5:
-            break
-        # get information from objects
-        ra, dec, period = float(row.ra), float(row.dec), float(row.period)
-
-        # get object id
-        response = api(
-            "GET", f"/api/sources?&ra={ra}&dec={dec}&radius={2/3600}", args.token
-        )
-        sleep(0.9)
-        data = response.json().get("data")
-        obj_id = None
-        if data["totalMatches"] > 0:
-            obj_id = data["sources"][0]["id"]
-        print(f"object {index} id:", obj_id)
 
-        # save new source
-        if obj_id is None:
-            obj_id = save_newsource(
-                gloria, group_ids, ra, dec, period=period, return_id=True
+    if (file == "parse") | (file == 'Parse') | (file == 'PARSE'):
```

Comment:
maybe if file in ["parse", 'Parse', 'PARSE']:

Reply:
Agreed, done!